assets/js/components/surveys/CurrentSurvey/utils.js

```diff
@@ -0,0 +1,192 @@
+/**
+ * CurrentSurvey utility functions.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Gets the current question and its ordinal.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object[]} questions The questions in the survey.
+ * @param {Object[]} answers   The answers to the questions.
+ * @return {Object} The current question and its ordinal.
+ */
+export function getCurrentQuestionAndOrdinal( questions, answers ) {
```

Comment:
I realised during development that `getCurrentQuestion()` as specced in the IB would need to return the ordinal as well, so renamed the function accordingly.